File: trendify.Server/trendify.Core/Models/Cart/CartItemModel.cs
namespace trendify.Core.Models.Cart
{
    public class CartItemModel
    {
        public int Id { get; set; }
        public int ProductId { get; set; }
        public string ProductName { get; set; } = null!;
        public string ImageUrl { get; set; } = null!;
        public decimal Price { get; set; }
        public int Quantity { get; set; }
    }
}

File: trendify.Client/src/app/models/cart/cartItem-model.ts
export class CartItemModel {
  id!: number;
  productId!: number;
  productName!: string;
  imageUrl!: string;
  price!: number;
  quantity!: number;

  constructor(init?: Partial<CartItemModel>) {
    if (init) {
      Object.assign(this, init);
    }
  }
}
